ayushma/models/users.py

```diff
@@ -27,6 +45,39 @@ class User(AbstractUser):
     allow_key = models.BooleanField(default=False)
 
     USERNAME_FIELD = "email"
-    REQUIRED_FIELDS = ["username"]
+    REQUIRED_FIELDS = ["email", "first_name"]
```

Comment:
```suggestion
    REQUIRED_FIELDS = ["username"]
```
No need to change this, username is configured to be the email.